bundle/edu.gemini.ags/src/test/scala/edu/gemini/ags/impl/SingleProbeStrategySpec.scala

```diff
@@ -39,7 +37,7 @@ import scala.collection.JavaConverters._
 import scalaz._
 import Scalaz._
 
-class SingleProbeStrategySpec extends Specification with NoTimeConversions {
```

Comment:
`NoTimeConversions` was a specs2 trait that avoided conflicts among implicit methods in specs2 and `scala.concurrent.duration._`. This PR upgrades specs2 to 3.7.1 and in that version there is no need for `NoTimeConversions`